integration/kubernetes/helm-chart/alluxio/Chart.yaml

```diff
@@ -12,7 +12,7 @@
 name: alluxio
 apiVersion: v1
 description: Open source data orchestration for analytics and machine learning in any cloud.
-version: 0.5.9
+version: 0.6.0
```

Comment:
not sure what's the convention here, is more common to have 0.5.10 or 0.6.0?
@cheyang @jiacheliu3

Reply:
We don't really have a convention on the versioning here. I'm fine with bumping to 0.6.0 as this is relatively stable now.